guardian/manage-frontend
# Continue Reading The Guardian component

If a user has gone through the switch journey on desktop or via a mobile but **not** using the app, we want to show a 'continue reading the guardian' button to direct them back to the homepage of The Guardian.

## What does this change?
A new button has been added to the switch complete screen which when clicked directs the user to the home page of The Guardian. The button should only be visible on desktop/mobile  **if**  user hasn't come via the app



## How to test
The new button should appear in the Switch Complete Storybook test. Since this remains behind a feature switch it can't be tested in production but on CODE (if the feature flag has been lifted) follow the product switch journey and you will see the button on the switch complete page.



## Images


From testing in CODE:
![image](https://user-images.githubusercontent.com/115992455/216961631-b43c713d-08ed-46b8-9641-e04b41eb2bd3.png)

## Accessibility
